taxi/models.py

```diff
@@ -0,0 +1,26 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
```

Comment:
I guess you need to realize str magic method, so your manufacturer will be as name, not "Manufacturer object (1)" (I mean your first screenshot)
Also, I would prefer to realize str in every model

Reply:
Thanks, fixed it :)